docs/developers/how-to/testing.rst

```diff
@@ -0,0 +1,103 @@
+=============
+Running tests
+=============
+
+MicroOVN has two types of tests, linter checks and functional tests.
+
+Linter checks
+-------------
+
+Linting is currently applied only to the tests themselves, not the main
+codebase. The prerequisites for running linter are:
+
+* ``make``
+* ``shellcheck``
+
+To perform linting, run following command:
+
+.. code-block:: none
+
+   make check-lint
```

Comment:
in the previous how-to we stated a sentence explaining what the cwd should be. we should be consistent